New option for Modification allowed settings in Registration (#4740)
indico/indico: CHANGES.rst

Improvements
^^^^^^^^^^^^

> - New option of Modification allowed 'Until approved' in Event's Registration settings (:pr:`4740`, thanks :user:`vasantvohra`)

Reviewer: ```suggestion
- Add a new "Until approved" option for a registration form's "Modification allowed"
  setting (:pr:`4740`, thanks :user:`vasantvohra`)
```

Improvements
^^^^^^^^^^^^

> 

Reviewer: can you please restore this empty line? :)
